from0k2bp.tex

```diff
@@ -292,9 +295,9 @@
 form}{A Pedersen commitment in elliptic curve form}}\label{a-pedersen-commitment-in-elliptic-curve-form}}
 
 Instead of using a hash function, we will use a point on an elliptic
-curve to achieve the same goal; the one way function here is scalar
+curve to achieve the same goal; the one-way function here is scalar
 multiplication of curve points:
-\[C = rH + aG \]
+\[C := rH + aG \]
```

Comment:
I don't think this is appropriate. While `:=` is standard for programming stuff, you will basically never see it in cryptography papers, which are presenting mathematical constructs, and mathematicians don't use this notation (I can accept that it might be helpful if they did, sometimes; indeed the misuse of `=` where the three line alternative (indicating identity, rather than just equality) is more correct, is a bit of a bugbear. But best stick with standard notation.

Reply:
Too bad. I find it helpful. I'll revert it.

Reply:
Reverted.